src/lib/x509/certstor_system_macos/certstor_macos.cpp

```diff
@@ -0,0 +1,475 @@
+/*
+* Certificate Store
+* (C) 1999-2019 Jack Lloyd
+* (C) 2019      René Meusel
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/build.h>
+
+#if defined(BOTAN_TARGET_OS_IS_DARWIN)
```

Comment:
No need for this - if the file is being compiled we know (by virtue of restrictions set in `info.txt`) that we are on OSX